lib/internal/process/promises.js

```diff
@@ -124,11 +126,18 @@ function promiseRejectHandler(type, promise, reason) {
   }
 }
 
+const multipleResolvesDeprecate = deprecate(
+  () => {},
```

Comment:
```suggestion
  () => { process.emit('multipleResolves', type, promise, reason); },
```

Reply:
No because we want to emit the warning only if someone is listening to `multipleResolves` - otherwise all users get the deprecation warning even if they don't use the feature.